examples/1.blockade/main.jl

```diff
@@ -4,15 +4,15 @@
 # By including a term in the effective Hamiltonian which adds an interaction between adjacent atoms in the Rydberg states, 
 # one can construct gates and nontrivial dynamics that create entanglement and correlation across the system.
 
-# The Rydberg interaction is a van der Waals interaction, where there is an energy shift if two adjacent atoms are both in the Rydberg state. 
+# The Rydberg interaction is a Van der Waals interaction, where there is an energy shift if two adjacent atoms are both in the Rydberg state. 
```

Comment:
Looking at Wikipedia (https://en.wikipedia.org/wiki/Van_der_Waals_force), it looks like van der Waals force?

Reply:
You're right! Checked Brittanica too, will be more careful in the future.